Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1934,7 +1934,9 @@ ExceptionOr<Ref<CanvasGradient>> CanvasRenderingContext2DBase::createConicGradie
     if (!std::isfinite(angleInRadians) || !std::isfinite(x) || !std::isfinite(y))
         return Exception { NotSupportedError };
 
-    return CanvasGradient::create(FloatPoint(x, y), angleInRadians, *this);
+    // Angle starts from x-axis for consistency within canvas methods https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-createconicgradientΩ
+    angleInRadians += piFloat / 2;
```

Comment:
This would result in overflow if `angleInRadians` is greater than `std::numeric_limits<float>::max() - π/2`. Is this input possible? If so, we should normalize the angle.

There is existing logic in `computeArcBounds` in `Path.cpp` to normalize an angle. That could be factored into a method in `GeometryUtilities.h`.